### PR TITLE
COMMERCE-6721 We should use a search container for passing orders to the orders portlets

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java [hunk 1]
```diff
@@ -487,6 +464,51 @@ public PortletURL getPortletURL() throws PortalException {
...
+				_searchContainer.getStart(), _searchContainer.getEnd());
+
+			commerceOrdersTotal =
+				_commerceOrderService.getUserPlacedCommerceOrdersCount(
+					_cpRequestHelper.getCompanyId(),
+					_cpRequestHelper.getChannelGroupId(), keywords);
+		}
+
+		_searchContainer.setTotal((int)commerceOrdersTotal);
+		_searchContainer.setResults(commerceOrders);
```
Comment (dsitu) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java:507]: I think we sort this.
```suggestion
		_searchContainer.setResults(commerceOrders);
		_searchContainer.setTotal((int)commerceOrdersTotal);
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Review by dsitu — APPROVED** [review]

**Comment by alecsloan** [comment]
ci:test:sf

**Comment by alecsloan** [comment]
ci:test:relevant

**Review by ivicac — APPROVED** [review]

**Comment by ivicac** [comment]
ci:forward

**Pull request closed without merging**